.github/workflows/automatic-release-to-main-merger.yml

```diff
@@ -72,7 +106,7 @@ jobs:
             gh pr close $id -d
           done <pr_ids.txt
         env:
-            GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+          GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
 
       - name: Notify Slack when closing of outdated merge-PRs failed 💬
         if: failure()
```

Comment:
this will also run if `create-new-branch` fails, right?

Reply:
Oh yeah good catch, I'll add an extra condition for the step above it if that fails.

Reply:
Updated: https://github.com/RasaHQ/rasa/pull/12293/commits/6628edb3db473a467459ee9136be3b7549a18f17